priv/repo/migrations/20220908132321_add_voice_text_chat.exs

```diff
@@ -0,0 +1,45 @@
+defmodule Ret.Repo.Migrations.AddFlyPermission do
+  use Ecto.Migration
+
+  def up do
+    alter table("hubs") do
+      modify(:member_permissions, :integer,
+        default:
+          %{
+            spawn_and_move_media: true,
+            spawn_camera: true,
+            spawn_drawing: true,
+            pin_objects: true,
+            spawn_emoji: true,
+            fly: true,
+            voice_chat: true,
+            text_chat: true
+          }
+          |> Ret.Hub.member_permissions_to_int()
+      )
+    end
+
+    execute("UPDATE ret0.hubs SET member_permissions = member_permissions | 192;")
+  end
+
+  def down do
+    # Probably don't ever want to revert this migration, but in case we do:
```

Comment:
We should at least be able to test rolling this data migration back. Assuming no data is lost either direction, why not uncomment it?

Reply:
I saw that other migrations had it disabled so I wasn't sure if there was a reason for it. I've re-enabled it.